### PR TITLE
[DependencyInjection] EnvVarProcessor - adding section about datetime prefix

### PR DESCRIPTION
**Pull request by chr-hertel**
belongs to symfony/symfony#34634 and symfony/symfony#34652

### CODE REVIEW

**Comment by OskarStark** [comment]
Hi Christopher 👋 
This contribution is really appreciated. Could you find out in which version it was merged to add a corresponding `versionadded` directive and target the right branch?

Thanks 👍

**Comment by chr-hertel** [comment]
i think this is going to be master, in case it gets merged ;)

**Comment by OskarStark** [comment]
Ah sorry I missed the CodePR, I only clicked the issue!

File: configuration/env_var_processors.rst [hunk 1]
```diff
@@ -604,6 +604,57 @@ Symfony provides the following env var processors:
...
+            use App\Service\Clock
+
+            // if NOW is not set, the DateTime will be using system time
+            $container->getDefinition(Clock::class)
+                ->setArgument('$now', '%env(datetime:default::NOW)%');
+
+    .. note::
+
+        It is also possible to use ``DateTimeImmutable`` instead by
+        using ``datetime_immutable`` as prefix.
```
Comment (OskarStark) [inline configuration/env_var_processors.rst]:
```suggestion
        using ``datetime_immutable`` as prefix.
        
    .. versionadded:: 5.1
    
        The ``datetime`` and ``datetime_immutable`` processors were introduced in Symfony 5.1.
```

**Comment by chr-hertel** [comment]
PRs is closed, closing here as well

**Pull request closed without merging**